spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/builder/ExportCdn.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2009 - 2022 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.spec.bootstrap.data.builder;
+
+public class ExportCdn {
```

Comment:
Could the autogenerated CdnDTO not be used instead of this?

Reply:
It could but it would have to carry webUrl in one of the unrelated properties such as name which I found confusing.